Sources/SwiftSyntax/SyntaxParser.swift

```diff
@@ -33,10 +33,15 @@ public enum ParserError: Error, CustomStringConvertible {
   /// normal circumstances, and it should be reported as a bug.
   case invalidSyntaxData
 
+  /// The SwiftSyntax parser library isn't compatible with this client
+  case hashVerificationFailed
```

Comment:
Recommend to name this `parserCompatibilityCheckFailed`.

Reply:
Done